.github/workflows/close-stale-issues.yml

```diff
@@ -12,25 +12,25 @@ env:
     and it is being closed. If you feel closing this issue is not the
     right thing to do, please leave a comment.
   WARN_MESSAGE: >
-    There has been no activity on this issue for
-    3 years and it may no longer be relevant.
-    It will be closed 1 month after the last non-automated comment.
+    There has been no activity on this issue for 90 days.
+    The help repository works best when sustained engagement moves conversation forward.
+    The issue will be closed 1 month after the last non-automated comment.
+    If you are still experiencing this issue on the latest supported versions of Node.js, please leave a comment.
 # yamllint enable
 
 jobs:
   stale:
     if: github.repository == 'nodejs/help'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/stale@v4
+      - uses: actions/stale@a20b814fb01b71def3bd6f56e7494d667ddf28da #4.1.1
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          # 3 years. this number is chosen to target around 25 initial issues, with then a natural flow as time progresses
-          days-before-stale: 1095
+          days-before-stale: 90
```

Comment:
open to other numbers...but in total this means that no activity in a THIRD of a year will close the issue. I think that sounds like a fair balance of empathy toward maintainer and asker time/schedules.

During the collab summit I also suggested multiple PRs that go from 3 years, to 2 years, to 1 year, to 6 months, or something like that... but it sounded good on paper and didn't land well with the audience...the only benefit is shortening what will be many notifications. Node.js collaborators are already used to a firehose and should have optimized the experience to their needs.